lddtree.sh

```diff
@@ -69,9 +68,11 @@ elf_specs_scanelf() {
 		sed -r 's: (LINUX|GNU)$: NONE:'
 }
 
+
+
 # functions for binutils backend
 elf_rpath_binutils() {
-	objdump -x "$@" | awk '$1 == "RPATH" || $1 == "RUNPATH" { print $2 }'
+	readelf -d "$1" | awk '$2 == "(RPATH)" || $2 == "(RUNPATH)" { print $5 }' | cut -d '[' -f 2 | sed 's/]//'
```

Comment:
- why readelf instead of objdump?
- why not do the string formatting directly in the first `awk` instead of spawning a `cut` and a `sed` process?